habitat_sim/utils/data/pose_extractor.py

```diff
@@ -28,8 +29,10 @@ def __init__(self, topdown_views, sim, pixels_per_meter=0.1):
         self.sim = sim
         self.pixels_per_meter = pixels_per_meter
 
-    def extract_poses(self, labels):
-        r"""Returns a numpy array of camera poses.
+    def extract_poses(self, labels, extraction_method):
+        r"""Returns a numpy array of camera poses. If extraction method is "closest", this method will
+        use bfs to find the closest point of interest to each camera position. If the extraction method
+        is "panorama", it will extract a full panorama of camera poses for each camera position
```

Comment:
When you mention panorama here, does that mean one image or a number of images taking turns? Let's make it explicit what to expect.